src/router.ts

```diff
@@ -0,0 +1,26 @@
+import { Router } from 'express';
+import { InvalidArgument } from './utils/errors/client.error';
+import UserController from './user/user.controller';
+import { wrapAsync } from './utils/wrappers';
+
+const appRouter = Router();
+
+appRouter.get('/isAlive', (req, res) => {
+  res.status(200).send('alive');
+});
+
+appRouter.get('/api/users/:id', wrapAsync(async (req, res) => {
```

Comment:
I think that the api-gateway should direct the routes to users, groups, etc. (I also think that because the group-service's routes don't start like that it should match). I would change the routes to '/:id' and '/'.